metadata.json

```diff
@@ -15,6 +15,14 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 4.13.0 < 6.0.0"
+    },
+    {
+      "name": "puppetlabs/apt",
+      "version_requirement": ">= 6.3.0 < 7.0.0"
+    },
+    {
+      "name": "puppetlabs-docker",
```

Comment:
I think the docker module isn't always used in this module, so it's a soft dependency. Please don't add it to metadata.json.

Reply:
Yes, you're right. I'll remove it righ away.